src/dex/fluid-dex/fluid-dex-e2e.test.ts

```diff
@@ -283,9 +240,9 @@ const limitsWide: DexLimits = {
   },
 };
 
-const ErrInsufficientBorrowable = new Error('insufficient borrowable');
-const ErrInsufficientMaxPrice = new Error('insufficient max price');
-const ErrInsufficientReserve = new Error('insufficient reserve');
+const ErrInsufficientBorrowable = new Error('DebtLimitReached');
+const ErrInsufficientMaxPrice = new Error('OracleUpdateHugeSwapDiff');
+const ErrInsufficientReserve = new Error('DebtReservesTooLow');
```

Comment:
assume we should check that `swapInAdjusted` returns `0n` instead of throwing error in tests

Reply:
done, those checks were just never done in catch, cause no errors had been thrown